cmd/node_agent/na/gcp.go

```diff
@@ -15,20 +15,28 @@
 package na
 
 import (
+	"fmt"
+
 	"cloud.google.com/go/compute/metadata"
 	"github.com/golang/glog"
 	"golang.org/x/net/context"
 	"google.golang.org/grpc"
+	"google.golang.org/grpc/credentials"
 	cred "istio.io/auth/pkg/credential"
 )
 
+const (
+	bearerTokenScheme = "Bearer"
+	httpAuthHeader    = "authorization"
+)
+
 type jwtAccess struct {
 	token string
 }
 
 func (j *jwtAccess) GetRequestMetadata(ctx context.Context, uri ...string) (map[string]string, error) {
```

Comment:
Where is this used?

Reply:
This is used by gRPC internally when making a connection.

Reply:
I see. Thanks.